tests/projects/base_pipeline.py

```diff
@@ -0,0 +1,25 @@
+import pathlib
```

Comment:
I assume you wanted this file to be the base for both the local and remote pipeline test classes but forgot to use it in the local one ?